### PR TITLE
Factominer

### PR DESCRIPTION
**Pull request by tina-r**
* [ ] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-factominer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
```
Comment (bgruening) [inline recipes/r-factominer/meta.yaml]: You are missing the checksum here @tina-r.

File: recipes/r-factominer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    - https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
+
+
+  # You can add a hash for the file here, like md5 or sha1
+  # md5: 49448ba4863157652311cc5ea4fea3ea
+  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
+  # patches:
```
Comment (bgruening) [inline recipes/r-factominer/meta.yaml]: Can you remove this comments?

File: recipes/r-factominer/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+# The original CRAN metadata for this package was:
+
+# Package: FactoMineR
+# Version: 1.35
+# Date: 2017-02-20
+# Title: Multivariate Exploratory Data Analysis and Data Mining
+# Author: Francois Husson, Julie Josse, Sebastien Le, Jeremy Mazet
+# Maintainer: Francois Husson <francois.husson@agrocampus-ouest.fr>
+# Depends: R (>= 3.3.0)
```
Comment (bgruening) [inline recipes/r-factominer/meta.yaml]: Can you remove this comment please?

**Comment by druvus** [comment]
The tool was added in #4801 so I will close this.

**Pull request closed without merging**